src/Domain/Item/ItemService.ts

```diff
@@ -343,4 +349,27 @@ export class ItemService implements ItemServiceInterface {
 
     return dto.items
   }
+
+  private async computeItemIdsToFetchBasedOnTransferLimit(itemQuery: ItemQuery): Promise<Array<string>> {
+    const itemUuidsToFetch = []
+    const itemContentSizes = await this.itemRepository.findContentSizeForComputingTransferLimit(itemQuery)
```

Comment:
`itemContentSizes` would be sorted/ordered right?

Reply:
yes